File: is28/sharutin28/CppTasks/1.cpp
#include <string>
#include <iostream>

using namespace std;

struct name_age
{
	int age;
	string name;
};

int main()
{
	struct name_age X[3];

	for(int i = 0; i< 3 ; i++)
	{
		cout << "как вас зовóт?";
		cin >> X[i].name;

		cout << "сколько вам лет?";
		cin >> X[i].age;
	}

	for(int i = 0; i< 3 ; i++)
	{
		cout <<"привет" << X[i].name << endl;
		cout <<"через 10 лет вам бóдет " << X[i].age+10 << endl;
	}

	return 0;
}

File: is28/zhirnova28/Python/zad6/1.py
def vvod():
 f=open('mass.txt','r')
 a=[]
 v = {}
 for i in range(2):
  p=f.readline()
  p1=p.split(':')
  v['name']=p1[0]
  v['grp']=p1[1]
  v['age']=int(p1[2])
  a.append(v.copy())
 return a
 
def res():
 print(a)

a=vvod()
vyvod=res()

